### PR TITLE
Support logo configuration

### LINKED ISSUE #941: [Portal, Appearance] Banner and SSO buttons configuration
- [x] [Move Appearance into its own tab](https://github.com/authgear/authgear-server/issues/1016)
- [x] [Revamp language widget](https://github.com/authgear/authgear-server/issues/1021)
- [x] [Add shorthand for app name, privacy policy link, terms of service link, default uris](https://github.com/authgear/authgear-server/issues/1030)
- [ ] [Enhance logo configuration](https://github.com/authgear/authgear-server/issues/1033)

---

> The following is the original description

---

- Full width
- Fixed Height

image

**Width and Height Option**

- SSO button shape
image

### LINKED ISSUE #1033: Add logo configuration

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #1033 
fix #941

### CODE REVIEW

File: portal/src/ThemePresetWidget.tsx [hunk 1]
```diff
@@ -161,7 +164,7 @@ function ThemePreset(props: ThemePresetProps) {
   return (
     <div
       className={cn(styles.preset, isSelected && styles.selected)}
-      onClick={onClick}
+      onClick={disabled ? undefined : onClick}
```
Comment (carmenlau) [inline portal/src/ThemePresetWidget.tsx:167]: Better to update the css `cursor` not to use `pointer` when it is disabled, the default it `auto`.

**Review by carmenlau — COMMENTED** [review]
A question about the banner setting, seems the **Left and Right Margin** is not very useful in both **Fixed height** and **Fixed width** setting?

File: portal/src/ThemePresetWidget.tsx (hunk 1 above)
Comment (louischan-oursky) [reply portal/src/ThemePresetWidget.tsx:171]: Add disabled state.

**Comment by louischan-oursky** [comment]
I confirmed with Frank and it should be horizontal padding with 100% width by default.

**Pull request merged**